Client/mods/deathmatch/logic/luadefs/CLuaEngineDefs.cpp

```diff
@@ -118,6 +132,26 @@ void CLuaEngineDefs::AddEngineTxdClass(lua_State* luaVM)
     lua_registerclass(luaVM, "EngineTXD", "Element");
 }
 
+void CLuaEngineDefs::AddEngineImgClass(lua_State* luaVM)
+{
+    lua_newclass(luaVM);
+
+    lua_classfunction(luaVM, "create", "engineLoadIMG");
+    lua_classfunction(luaVM, "add", "engineAddImage");
+    lua_classfunction(luaVM, "remove", "engineRemoveImage");
+    lua_classfunction(luaVM, "getFile", "engineImageGetFile");
+    lua_classfunction(luaVM, "getFileList", "engineImageGetFileList");
```

Comment:
Perhaps `getFiles` would be more correct.
For example its `getElementsByType` not `getElementListByType`.